hasher-matcher-actioner/webapp/src/Matches.jsx

```diff
@@ -75,189 +75,33 @@ export default function Matches() {
-                  <button
-                    type="button"
-                    className="btn btn-outline-primary btn-sm">
-                    Details
-                  </button>
-                </td>
-              </tr>
-              <tr>
-                <td className="align-middle">file6.jpg</td>
-                <td className="align-middle">
-                  652fe95ab4ae5129c07e92da787e525ab44b736ea5ab5ce4485ba344158b8e86
-                </td>
-                <td className="align-middle">5 Feb 2021 1:44pm</td>
-                <td className="align-middle">Positive</td>
-                <td className="align-middle">
-                  <button
-                    type="button"
-                    className="btn btn-outline-primary btn-sm">
-                    Details
-                  </button>
-                </td>
-              </tr>
-              <tr>
-                <td className="align-middle">file7.jpg</td>
-                <td className="align-middle">
-                  652fe95ab4ae5129c07e92da787e525ab44b736ea5ab5ce4485ba344158b8e86
-                </td>
-                <td className="align-middle">5 Feb 2021 1:44pm</td>
-                <td className="align-middle">Seen</td>
-                <td className="align-middle">
-                  <button
-                    type="button"
-                    className="btn btn-outline-primary btn-sm">
-                    Details
-                  </button>
-                </td>
-              </tr>
-            </tbody>
-          </table>
-        </div>
-      </div>
-      <MatchList2 />
+      <MatchList />
       <MatchDetailsModal show={showModal} onHide={() => setShowModal(false)} />
     </>
   );
 }
 
-function MatchDetailsModal(props) {
-  // TODO add prop typing
-  // eslint-disable-next-line react/prop-types
-  const {show, onHide} = props;
-  return (
-    <Modal
-      show={show}
-      onHide={onHide}
-      size="lg"
-      aria-labelledby="contained-modal-title-vcenter"
-      centered>
-      <Modal.Header closeButton>
-        <Modal.Title id="contained-modal-title-vcenter">
-          Match Details
-        </Modal.Title>
-      </Modal.Header>
-      <Modal.Body>
-        <h4>Centered Modal</h4>
-        <p>
-          Cras mattis consectetur purus sit amet fermentum. Cras justo odio,
-          dapibus ac facilisis in, egestas eget quam. Morbi leo risus, porta ac
-          consectetur ac, vestibulum at eros.
-        </p>
-      </Modal.Body>
-      <Modal.Footer>
-        <Button onClick={onHide}>Close</Button>
-      </Modal.Footer>
-    </Modal>
-  );
-}
-
-function MatchList2() {
+function MatchList() {
   const [matchesData, setMatchesData] = useState(null);
+  const [showModal, setShowModal] = useState(false);
 
   useEffect(() => {
     fetchMatches().then(matches => setMatchesData(matches.matches));
     // .catch(err => console.log(err));
   }, []);
 
+  function formatTimestamp(timestamp) {
```

Comment:
We haven't really had enough code written for module patterns to emerge... for now, how about creating a directory helpers or utils and create a file FormatTimestamp.js there, with this as the default export?

Reply:
yep, I created Utils.jsx in my next PR for this reason